userguide/content/en/docs/adding-content/lookandfeel.md

```diff
@@ -213,7 +214,33 @@ site's [configuration file][].
 [configuration file]: https://gohugo.io/getting-started/configuration/#configuration-file
 [primary color]: #site-colors
 
+### Styling your brand logo and name
```

Comment:
"your brand logo and name" -> "your project logo and name"

Reply:
Done